src/pages/contact_us/_offices.tsx

```diff
@@ -196,17 +220,62 @@ export const Offices = () => {
                     <ResHeader as="h2" type="page-title">
                         {localize('Our offices')}
                     </ResHeader>
+                    <ContinentHeader>{localize('EUROPE')}</ContinentHeader>
                     <GridLayout>
+                        <Flex fd="column" max_width="48.6rem" id="france">
+                            <OfficeHeader>
+                                <div>
+                                    <img src={France} alt="France - Eiffel Tower"/>
+                                </div>
+                                <Header as="h4" type="sub-section-title" mt="0.8rem" mb="1.6rem">
+                                    {localize('France')}
+                                </Header>
+                            </OfficeHeader>
+
+                            <BorderBox>
+                                <MapContainer>
+                                    <LocalizedLink
+                                        to="https://g.page/wojo-madeleine-coworking-paris-8?share"
+                                        external="true"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                    >
+                                        <QueryImage
+                                            alt="Deriv's office location map in France"
```

Comment:
`Deriv's office location map in France` or` Deriv's office location in France` ?